containers/newtask/Layout.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FunctionComponent } from 'react';
+import { StatusBar, TextInput, View } from 'react-native';
+import { Colors } from '../../colors/Colors';
+import { Strings } from '../../strings';
+import { styles } from './styles';
+import { LayoutProps } from './types';
+
+export const Layout: FunctionComponent<LayoutProps> = ({
+  firstInputColor,
+  secondInputColor,
+  onFocusTitle,
+  onBlurTitle,
+  onFocusDescription,
+  onBlurDescription,
+  onChangeTitle,
+  onChangeDescription
+}: LayoutProps) => {
+  return (
+    <View style={styles.container}>
+      <StatusBar backgroundColor={Colors.customBlue} />
+      <TextInput
+        onFocus={() => onFocusTitle()}
```

Comment:
In these cases you can avoid wrapping the function in an arrow, just pass in the function